Documentation/Storage-Configuration/Advanced/cephx-key-rotation.md

```diff
@@ -0,0 +1,144 @@
+---
+title: CephX Key Rotation
+---
+
+!!! attention
+    This feature is experimental.
+
+Rook is able to rotate [CephX authentication keys](https://docs.ceph.com/en/latest/dev/cephx/) used
+by Ceph daemons and clients.
+
+For this experimental feature, some caveats should be noted:
+
+- Only Ceph versions v19.2.3 and higher have the capabilities Rook requires for key rotation.
+- Ceph Monitor (mon) keys cannot be rotated in Ceph v19 due to Ceph architecture limitations.
+
+## Overview
```

Comment:
How about also pointing out that the key rotation is disabled by default?

Reply:
I updated the paragraph to this, which explains the default policy is `Disabled`. I hope that is still clear while also concise

> CephX keys can be rotated when desired on a one-off basis. To provide this capability, Rook utilizes
an approximation of Kubernetes's resource generation. A one-time key rotation is initiated by
specifying `KeyGeneration` as the desired policy (the default policy is `Disabled`) and also
specify a key generation higher than the current generation.